src/azure-cli/azure/cli/command_modules/vm/commands.py

```diff
@@ -288,6 +288,7 @@ def load_command_table(self, _):
         g.generic_update_command('update', setter_name='update_vm', setter_type=compute_custom, supports_no_wait=True)
         g.wait_command('wait', getter_name='get_instance_view', getter_type=compute_custom)
         g.custom_command('auto-shutdown', 'auto_shutdown_vm')
+        g.command('assess-patch', 'assess_patches')
```

Comment:
minor thing, why patch in argument, patches in function?

Reply:
btw, custom file missed in commit?

Reply:
Users don't want to type two more letters. assess_patches is a name defined in swagger.
No custom logic needed.

Reply:
it's a minor thing. while it's better to align with swagger. user could get the hint from name it's plurality

Reply:
OK